news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+# Work on the new theme has started
```

Comment:
The 1st sentence of the post is used as a summary inside the homepage https://build.prestashop.com/
so we need to write something that summarizes your article before the first h1

Also I think you need to use `##` and not `#` because the title of the article will be inserted by github as h1. So you only should put h2